poutyne/plot.py

```diff
@@ -0,0 +1,156 @@
+import os
+import itertools
+
+try:
+    import matplotlib.pyplot as plt
+    from matplotlib.ticker import MaxNLocator
+
+    matplotlib = True
+except ImportError:
+    matplotlib = False
+
+try:
+    import pandas as pd
+except ImportError:
+    pd = None
```

Comment:
Because pandas only has one namespace (`pd`) whereas matplotlib, you have `plt` and also other imported name such as `MaxNLocator` here. That's why I preferred to use a boolean here for matplotlib to avoid having code that doesn't check the right namespace.